addon/mixins/sortable-item.js

```diff
@@ -320,17 +321,18 @@ export default Mixin.create({
     if (this.get('isBusy')) { return; }
 
     let drag = this._makeDragHandler(event);
+    let dragThrottled = ev => throttle(this, drag, ev, 50, false);
```

Comment:
@scottmessinger I think we can also refresh the coordinates (x and y) every 50-100 milliseconds and make smooth animation using CSS transitions. In that case the delay would be visible only if the user would have change direction of dragging.